arho_feature_template/plugin.py

```diff
@@ -162,6 +164,24 @@ def initGui(self) -> None:  # noqa N802
             add_to_toolbar=True,
         )
 
+        self.validate_plan_action = self.add_action(
+            "",
+            text="Vahvista kaava",
+            triggered_callback=self.validate_plan,
+            add_to_menu=True,
+            add_to_toolbar=True,
+            status_tip="Vahvista kaava",
+        )
+
+        self.plugin_settings_action = self.add_action(
+            "",
+            text="Asetukset",
+            triggered_callback=self.open_settings,
+            add_to_menu=True,
+            add_to_toolbar=True,
```

Comment:
Riittää varmaan, että asetukset löytyvät plugin-menusta, ei tarvi olla toolbarissa.
```suggestion
            add_to_toolbar=False,
```